tensorflow/stream_executor/cuda/cuda_blas.cc

```diff
@@ -1861,12 +1861,11 @@ bool CUDABlas::DoBlasGemm(
 
   bool use_tensor_ops = false;
 #if CUDA_VERSION >= 9000
-  int cc_major, cc_minor;
-  stream->parent()->GetDeviceDescription().cuda_compute_capability(&cc_major,
-                                                                   &cc_minor);
+  DeviceVersion device_version;
+  stream->parent()->GetDeviceDescription().device_hardware_version();
```

Comment:
This does not look correct?  You mean to assign to the device_version variable, right?